lib/config/rules/stylistic-issues.js

```diff
@@ -147,6 +153,8 @@ module.exports = {
   'space-unary-ops': ['warn', {words: true, nonwords: false}],
   // Require or disallow a space immediately following the // or /* in a comment
   'spaced-comment': ['warn', 'always', {markers: ['=']}],
+  // Require or disallow spacing between template tags and their literals
+  'template-tag-spacing': 'off',
```

Comment:
My preference would be never having spacing.  If there's a space, my brain starts reading in poetry mode.